addons/sounds-duration/addon.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Sound Duration",
```

Comment:
```suggestion
  "name": "Sound duration display",
```

Reply:
I don't think we use "display" in any addon names... We don't call it clone counter display

Reply:
@TheColaber a counter implies a display